kubectl-minio/cmd/tenant-create.go

```diff
@@ -82,6 +82,7 @@ func newTenantCreateCmd(out io.Writer, errOut io.Writer) *cobra.Command {
 	f.BoolVar(&c.tenantOpts.DisableAntiAffinity, "enable-host-sharing", false, "[TESTING-ONLY] disable anti-affinity to allow pods to be co-located on a single node (unsupported in production environment)")
 	f.StringVar(&c.tenantOpts.KmsSecret, "kes-config", "", "name of secret for KES KMS setup, refer https://github.com/minio/operator/blob/master/examples/kes-secret.yaml")
 	f.BoolVar(&c.tenantOpts.EnableAuditLogs, "enable-audit-logs", true, "Enable/Disable audit logs")
+	f.BoolVar(&c.tenantOpts.EnableTLS, "enable-tls", true, "Enable/Disable TLS")
```

Comment:
It is simply easier to take `--disable-tls` as an option instead of `--enable-tls=false`  WDYT? @pjuarezd

Reply:
I am good with that, `--disable-tls` . It was a little tricky because default values are not a good inicator of whether the parameter was set or not by the user, change is pushed in latest commit